### DISCUSSION TITLE
.toObject() in onLoad loading elements of the previously opened workflow.

### DISCUSSION THREAD

**Discussion by kaushalendra-pandey** [xyflow/xyflow#2305] [General]
I have a project having multiple workflows and each workflow has its own set of node/edges i.e. elements. Now when I open a workflow, inside onLoad I am setting reactFlowInstance. But when I send this data to the backend, it sends the elements of the previously opened workflow. But the nodes and elements present on the canvas are the element of the current workflow. Is there a way to revalidate the reactFlowInstance we get in onLoad??

**Comment by moklick** [comment]
It might be easier to use the [`useReactFlow` hook](https://reactflow.dev/docs/api/hooks/use-react-flow/). If that doesn't solve your issue, could you create a codesandbox or repo for this issue?
